2/contrib/openshift/base-plugins.txt

```diff
@@ -17,8 +17,7 @@ htmlpublisher:1.21
 jira:3.0.17
 job-dsl:1.77
 junit:1.30
-kubernetes:1.29.7
-kubernetes-client-api:4.13.3-1
+kubernetes:1.30.0
```

Comment:
@akram I noticed in my recent testing there is a now a version 1.30.1 for the k8s plugin

perhaps not required, but something to go ahead and move to